app/tests/__init__.py

```diff
@@ -0,0 +1,17 @@
+import os
+import tempfile
+
+import pytest
+
+from app import app
+
+@pytest.fixture
+def client():
+    db_fd, app.config["DATABASE"] = tempfile.mkstemp()
+    app.config["TESTING"] = True
+
+    with app.test_client() as client:
+        yield client
+
+    os.close(db_fd)
+    os.unlink(app.config["DATABASE"])
```

Comment:
All test setup stuff can go in here. Overtime, we can build up a nice `TestCase` class full of all the helpers we want.